src/hooks/user/useUserScopeDocument.js

```diff
@@ -0,0 +1,15 @@
+import { doc, updateDoc } from "firebase/firestore";
+import { useDocument } from "react-firebase-hooks/firestore";
+import { firestore } from "../../firebase.config";
+import useAuthedUser from "./useAuthedUser";
+
+const FirestoreDocumentData = () => {
+  const [user, userLoading, userError] = useAuthedUser();
+  const docRef = user !== undefined ? doc(firestore, "users", user.uid) : undefined;
+  const [value, loading, error] = useDocument(docRef);
+  
+
+  return [docRef, value, loading || userLoading, error || userError];
+};
+
+export default FirestoreDocumentData;
```

Comment:
This does not seem to follow the naming rules of custom react https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hookhooks 
```
A custom Hook is a JavaScript function whose name starts with ”use” and that may call other Hooks.
```